src/test/java/io/jenkins/plugins/artifact_manager_jclouds/s3/S3BlobStoreConfigTest.java

```diff
@@ -7,6 +7,9 @@
 import java.util.logging.Logger;
 import jenkins.model.ArtifactManagerConfiguration;
 import static org.junit.Assert.*;
+
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.core.IsInstanceOf;
```

Comment:
Conventionally import these from `Matchers`.

Reply:
I'm not sure I understood your comment, do you mean I shouldn't use hamcrest and use something else instead?

Reply:
⇒ https://github.com/jenkinsci/artifact-manager-s3-plugin/pull/339#discussion_r1062568555